add panels to dash

Victor-M-Giraldo opened the pull request:
closes #1280

Victor-M-Giraldo commented:
How do we get these panels to actually display meaningful data and not just a static graph?

dyc3 commented:
It just requires changing the data source. Should be a trivial change for the most part. I can show the group when we meet next in person.

dyc3 commented on packages/ott-vis/provisioning/dashboards/dashboard.json:
```diff
@@ -18,38 +18,273 @@
+					"sort": "none"
 				}
-			],
-			"title": "Panel Title",
-			"type": "ott-vis-panel"
```
We need to keep this panel, ideally in the same spot it was before.

Victor-M-Giraldo replied:
Added it back. What is this one going to represent though? Didn't see it line up with anything in the doc which is why I deleted it.

dyc3 reviewed (changes requested):
This removes our custom panel, which we need for our development workflow.

dyc3 reviewed (changes requested):
Please lay them out better so you don't need to scroll to see the custom panel. The custom panel should be at the top.

Victor-M-Giraldo commented:
Moved them around. What's the custom panel going to represent? Is it the default/region views? I deleted it originally because I didn't see what it lined up with in the doc.

dyc3 commented:
Yes, its going to be the default/region views.
